master/buildbot/changes/gerritchangesource.py

```diff
@@ -298,17 +312,49 @@ def streamProcessStopped(self):
             self.startStreamProcess()
             self.streamProcessTimeout = self.STREAM_BACKOFF_MIN
 
+    def buildGerritCommand(self, *gerrit_args):
```

Comment:
is this a public method?  if not. use `_buildGerritCommand`

Reply:
Done